### PR TITLE
CSS-Brochure - Frank Bishop

### PR DESCRIPTION
**Pull request by FrankBishop**
Screenshot (31)

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  font-size: 10px;
+  margin-top: 9px;
+  margin-bottom: 1px;
+  margin-left: 780px;
+  text-align: left;
+}
+
+form {
+  text-align: center;
+}
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:20]: You probably don't meant to text align everything.

File: css-brochure-form/styles.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+input,
+option {
+  font-family: 'AtlasGrotesk', sans-serif;
+  width: 20%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:25]: You should stick to `px` for widths as much as possible.

File: css-brochure-form/styles.css [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+  border: solid;
+  border-radius: 10px;
+  border-color: #f2f2f2;
+  padding: 7px 12px 7px;
+  margin-bottom: 5px;
+}
+
+#submit {
+  width: 4%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:34]: You should stick to `px` for widths as much as possible.

File: css-brochure-form/styles.css [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  border-radius: 60px;
+  border: hidden;
+  font-family: 'Brandon Grotesque', sans-serif;
+  padding: 6px 3px 6px 3px;
+  margin-right: -310px;
+  text-align: center;
+}
+
+#howmany {
+  width: 20%;
```
Comment (codingcodymiller) [inline css-brochure-form/styles.css:46]: You should stick to `px` for widths as much as possible.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Hey Frank! Great job so far, please see my notes. I have recommended a few changes, and then we can adjust from there.

**Comment by FrankBishop** [comment]
Thanks for the feedback, here is an updated version that implemented the feedback.

Screenshot (45)

**Review by codingcodymiller — APPROVED** [review]
🥳 🥳 🥳

**Pull request merged**